extensions/meme_stats.py

```diff
@@ -30,7 +30,7 @@ async def main(ctx: lightbulb.Context) -> None:
     # todo: add a limit or aggregation of greater time periods or whatever
     # todo: get someone else to do this
     data = cursor.execute(f"""
-        select strftime('%Y-%m-%d', time_sent) as time_period,
+        select strftime('%Y-%m-%d', datetime(time_sent, '+10 hours')) as time_period,
             AVG(meme_score) as avg_meme_score
         from meme_stats
         where user = ? and time_sent >= date('now', '-1 {time_period}')
```

Comment:
While we're here could you please not template user-supplied values into SQL statements.

Reply:
I was lazy and a check seemed redundant with Lightbulb already validating, but as usual a good point, Kitti security must be maintained.